panels/dev-service/ha-panel-dev-service.html

```diff
@@ -1,28 +1,38 @@
-<link rel="import" href="../../bower_components/polymer/polymer.html">
```

Comment:
Sorry about that. I didn't think about it. I was just trying to clean it up. Next time I will do that in a seperate commit.